src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.ArrayList;
+
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sort the record in lexicographical order by  name.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    public SortCommand(){
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. missing space between `SortCommand()` and `{`.